### PR TITLE
GIX-2184: Change tokens pages title

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Add the token symbol in the title of the accounts page.

# Changes

* Use accountsTitleStore in the accounts layout.
* Change title in tokens layout for consistency.
* Remove unused i18n key.

# Tests

* Change and add a test in the accounts Layout spec file.
* Change test for tokens layout.

# Todos

- [ ] Add entry to changelog (if necessary).

### CODE REVIEW

**Comment by lmuntaner** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #4095** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/4095?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/nns-dapp/4095?utm_source=stack-comment).

File: frontend/src/routes/(app)/(nns)/tokens/+layout.svelte [hunk 1]
```diff
@@ -6,7 +6,7 @@
   import MainWrapper from "$lib/components/tokens/MainWrapper.svelte";
 </script>
 
-<LayoutList title={$i18n.tokens.title}>
+<LayoutList title={$i18n.navigation.tokens}>
```
Comment (lmuntaner) [inline frontend/src/routes/(app)/(nns)/tokens/+layout.svelte:9]: To be consistent with the accounts page where the title is "My <token> Tokens".

**Comment by lmuntaner** [comment]
UI:

**Comment by lmuntaner** [comment]
@dskloetd please review

**Review by dskloetd — APPROVED** [review]

**Pull request merged**